lib/routes/password.js

```diff
@@ -333,6 +333,7 @@ module.exports = function (
           .then(db.emailRecord.bind(db, email))
           .then(
             function (emailRecord) {
+              emailRecord.createdAt = undefined
```

Comment:
nit: Could probably add a comment here about why this value is being set to undefined.